api_test.go

```diff
@@ -90,3 +90,56 @@ func TestBezier(t *testing.T) {
 	gproc = proc.Proc
 	proc.Run(t)
 }
+
+func TestRandom(t *testing.T) {
+	old := gproc
+	defer func() {
+		gproc = old
+	}()
+
+	tests := []struct {
```

Comment:
perhaps add a call to an explicit seed seeding before going through these tests to ensure reproducibility (and robustness against adding new tests exercizing the PRNG) ?

Reply:
changed to explicit seed on a test local proc to make sure that tests do not interfere with each other